test/language/statements/class/subclass/class-definition-null-proto-missing-return-override.js

```diff
@@ -38,4 +38,4 @@ class Foo extends null {
 
 var foo = new Foo();
```

Comment:
See [step 15 of ClassDefinitionEvaluation](https://tc39.github.io/ecma262/#sec-runtime-semantics-classdefinitionevaluation), which is the only place in the spec that a constructor is set to `"derived"`, and which does not apply to classes which extend `null`.